man/qhavesex.1
.TH QHAVESEX 1 "05 February 2012" "qhavesex 0\&.a"
.SH NAME
qhavesex
.SH SYNOPSIS
.B qhavesex
[\-h] [\-dqv] [\-b \fIbackends\fR] ...
.br
.B qhavesex
[\-\-help] [\-\-version]

.SH DESCRIPTION
.LP

Qt application allowing to interact with various dating websites.

.SS Supported websites:
* aum (“Adopte un mec” french dating website)
.SH OPTIONS
.TP
\fB\-\-version\fR
show program's version number and exit
.TP
\fB\-h\fR, \fB\-\-help\fR
show this help message and exit
.TP
\fB\-b BACKENDS\fR, \fB\-\-backends=BACKENDS\fR
what backend(s) to enable (comma separated)

.SH LOGGING OPTIONS
.TP
\fB\-d\fR, \fB\-\-debug\fR
display debug messages
.TP
\fB\-q\fR, \fB\-\-quiet\fR
display only error messages
.TP
\fB\-v\fR, \fB\-\-verbose\fR
display info messages
.TP
\fB\-\-logging\-file=LOGGING_FILE\fR
file to save logs
.TP
\fB\-a\fR, \fB\-\-save\-responses\fR
save every response

.SH COPYRIGHT
Copyright(C) 2010-2011 Romain Bignon
.LP
For full COPYRIGHT see COPYING file with weboob package.
.LP
.RE
.SH FILES
"~/.weboob/backends" 

.SH SEE ALSO
Home page: http://weboob.org/Qhavesex


ChangeLog
Weboob 0.a (2012-02-05)
	General
	* New backend: Boursorama (CapBank).
	* New backend: CMB (CapBank).
	* New backend: HSBC (CapBank).
	* New backend: ING (CapBank).
	* New backend: Nolifetv (CapVideo).
	* New backend: RadioFrance (CapVideo, CapRadio) which merges
	  FranceInter and LeMouv backends.
	* Support repositories to manage backends (#747).
	* Support XDG Base Directory Specification (#765).
	* Make CapCollection understandable and usable by humans.

	Console applications
	* Ability to add all supported modules at once.
	* With -a, try to separate saved responses by backend (#661).

	Qt applications
	* Verbose error messages (#775).

	Backend: AuM
	* Display the 'title' field on contact profiles.
	* Get events (baskets, charms and visits).
	* Add city in status message.
	* Add 'Sent with my iPhone' in mail signatures when needed.
	* Display at least one photo url in profile even if there isn't any
	  visible.
	* Add a 'ratio' field.
	* Fix: getting contacts list.
	* Fix: support new API.
	* Fix: ipaddr is not available anymore.
	* Fix: do not crash if there are php errors before json.

	Backend: Batoto
	* Change .com to .net.

	Backend: BNP
	* Fix: website changed (#769).

	Backend: BP
	* Fix: website changed.
	* Fix: open images in memory instead of saving them in temp files.

	Backend: CanalPlus
	* Support https?://www\.canal-?plus\.fr/.*\?vid=(\d+) urls.

	Backend: CreditMutuel
	* Internal transfer implemented.
	* Fix: website changed.

	Backend: Dailymotion
	* Fix: support missing descriptions.

	Backend: DLFP
	* Limit feed entries by date instead of number of results.

	Backend: FranceTelevisions
	* Fix: website changed.

	Backend: Gazelle
	* Handle general errors when website is unavailable.

	Backend: INA
	* Fix: parsing of URLs.

	Backend: Kickass
	* Fix: website changed.

	Backend: LCL
	* Add support for various types of accounts.
	* List 45 days history by default.
	* Fix: website changed.

	Backend: LeFigaro
	* Better cleanup of generic articles (#725).
	* Fix: website changed.

	Backend: Orange
	* Fix: unicode errors.

	Backend: Pastebin
	* Handle user pages.

	Backend: Piratebay
	* Fix: website changed.

	Backend: SocieteGenerale
	* Fix: website changed.

	Backend: Youjizz
	* Fix: video URLs have changed.

	Backend: Youporn
	* Fix: website changed.

	Backend: Youtube
	* Fix: website changed.

	Application: QBoobMsg
	* When a thread is a discussion, display messages linearly.
	* Add a button 'Profile'.

	Application: QHaveSex
	* Add a 'Events' tab.
	* Add a 'Notes' tab for taking notes about a contact.

	Application: QVideoob
	* Fix: crash when there is no thumbnail on a video (#708).

	Application: weboob-config-qt
	* Ability to edit sources.list, update repositories and install
	  modules.

Weboob 0.9 (2011-10-10)
	General
	* New backend: Batoto (CapGallery).
	* New backend: Eatmanga (CapGallery).
	* New backend: FranceTelevisions (CapVideo).
	* New backend: HDS (CapMessages).
	* New backend: Izneo (CapGallery).
	* New backend: Mangafox (CapGallery).
	* New backend: Mangahere (CapGallery).
	* New backend: Mangareader (CapGallery).
	* New backend: Mangatoshokan (CapGallery).
	* New backend: Nova (CapRadio).
	* New backend: PhpBB (CapMessages, CapMessagesReply).
	* New backend: Simplyread.it (CapGallery).
	* New application: boobtracker (CapBugTracker).
	* New script: hds/export.py to export stories from
	  histoires-de-sexe.net into a sqlite database.
	* License change to AGPLv3+.
	* Ability to not save backend passwords. Applications will prompt them
	  when needed.

	Backend: Arte
	* Set duration on video results.

	Backend: AuM
	* Rewrite to use the mobile API.

	Backend: BNPorc
	* Get operation categories.

	Backend: CrAgr
	* Get full history.
	* Optimized operations retrieval for CA Centre.
	* Fix: parsing accounts list.

	Backend: CreditMutuel
	* Maine-Anjou, Basse-Normandie bank added.

	Backend: Dailymotion
	* Fix: parsing duration.
	* Fix: parsing of lives in search results.

	Backend: DLFP
	* Fix: disconnection URL needs to be called with POST instead of GET.
	* Fix: do tests on alpha.linuxfr.org.
	* Fix: relevance on news/diaries.

	Backend: Gazelle
	* Handle login errors.
	* Fix: parsing on several gazelle websites.

	Backend: INA
	* Support videos on www.ina.fr (in addition to boutique.ina.fr).

	Backend: IPInfoDB
	* Fix: no crash when there are no coordinates from server.

	Backend: Kickass
	* Fix: parsing of search results (#663).

	Backend: MediaWiki
	* Handle errors from API.

	Backend: MeteoFrance
	* Fix: website changes.

	Backend: Orange
	* Fix: handling of pages.

	Backend: PirateBay
	* Fix: parsing of search results.

	Backend: Redmine
	* Now implements CapBugTracker (#684).

	Backend: Transilien
	* Supports the roadmap feature (#681).

	Backend: Youtube
	* Support more URLs.
	* Support authentication.
	* Fix: website changes.

	Application: boobank
	* Add the 'qif' formatter.

	Application: boobank-munin
	* When handling an incorrect password error, disable backend to
	  prevent your bank to disable your account (for example with BNP).

	Application: pastoob
	* Ability to pipe output.

	Application: traveloob
	* New command 'roadmap' to get the roadmap between to stations. It's
	  also possible to give a departure or an arrival time (#681).

	Application: videoob
	* Do not run player in background (#664).
	* Command 'download' supports mms streams.

	Application: weboorrents
	* The given filename to command 'getfile' is optional (#683).

Weboob 0.8 (2011-05-08)
	General
	* New backend: E-Hentai (CapGallery).
	* New backend: FranceInter (CapRadio).
	* New backend: LeMouv (CapRadio).
	* New backend: PasteALaCon (CapPaste).
	* New backend: PasteBin (CapPaste).
	* New application: Boobathon (CapContent).
	* New application: Galleroob (CapGallery).
	* New application: Pastoob (CapPaste).
	* Fix lot of unicode problems.
	* License changed to AGPLv3+.
	* FreeBSD support.

	Repl applications

tools/packaging/setup.py.d/qhavesex.py
#! /usr/bin/env python
# -*- coding: utf-8 -*-

# Copyright(C) 2010  Christophe Benz
#
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, version 3 of the License.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program; if not, write to the Free Software
# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.


from setuptools import setup

import os


os.system('make -C weboob/applications/qhavesex/ui')

setup(
    name='weboob-qhavesex',
    version='0.1',
    description='QHaveSex, the Weboob sexual life swiss-knife, Qt version',
    long_description='Optimize your probabilities to have sex on dating websites',
    author='Romain Bignon',
    author_email='weboob@lists.symlink.me',
    maintainer='Christophe Benz',
    maintainer_email='christophe.benz@gmail.com',
    license='GPLv3',
    url='http://www.weboob.org/QHaveSex',
    namespace_packages = ['weboob', 'weboob.applications'],
    packages=[
        'weboob',
        'weboob.applications',
        'weboob.applications.qhavesex',
        'weboob.applications.qhavesex.ui',
        ],
    scripts=[
        'scripts/qhavesex',
        ],
    install_requires=[
        'weboob-core-qt', # python-weboob-core-qt
        'weboob-backends-dating', # python-weboob-backends-dating
        ],
)
